core/src/main/scala/org/ensime/core/debug/StructureConverter.scala

```diff
@@ -167,7 +167,7 @@ class StructureConverter(private val sourceMap: SourceMap) {
     import org.ensime.util.file._
     val pcLocation = sourceMap.newLineSourcePosition(frame.location).getOrElse(
       LineSourcePosition(
-        File(frame.location.sourcePath).canon,
+        File(frame.location.sourcePath).canon.toPath,
```

Comment:
You mean canonising both `jar:file:` pattern and the normal path differently ? Wouldn't `canon` do it for us for both the cases ?